### PR TITLE
Add dashboards

### PR DESCRIPTION
**Pull request by niqdev**
* [kubernetes-dashboard](https://github.com/kubernetes/dashboard)
* [kube-ops-view](https://github.com/hjacobs/kube-ops-view)

### CODE REVIEW

**Comment by conorfennell** [comment]
Update the README to explain the two applications.

**Review by conorfennell — APPROVED** [review]

File: applications/templates/kube-system/spotify-docker-gc.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ kind: Application
 metadata:
   name: spotify-docker-gc
 spec:
-  project: kube-system
+  project: kube-system-project
```
Comment (conorfennell) [inline applications/templates/kube-system/spotify-docker-gc.yaml]: kube-system-project???

File: applications/templates/kube-system/metrics-server.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@ kind: Application
 metadata:
   name: metrics-server
 spec:
-  project: kube-system
+  project: kube-system-project
```
Comment (conorfennell) [inline applications/templates/kube-system/metrics-server.yaml]: metrics server is an extension of the kubernetes api, kube-system makes sense for it

File: applications/templates/observe/kube-ops-view.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: kube-ops-view
+spec:
+  project: observe-project
```
Comment (conorfennell) [inline applications/templates/observe/kube-ops-view.yaml]: it is implicitly a project, why sub call it project

I missed this when you were doing it. 

you would not call the application name `kube-ops-view-application`

Comment (niqdev) [reply applications/templates/observe/kube-ops-view.yaml]: At the beginning I added that prefix for clarity, but I do agree that it doesn't make sense anymore. In this PR I've added it just for consistency and I was planning to make a separate PR to remove them all at once

**Pull request merged**